understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
                   </p>
                   
                   
-                  <p>The difference between the contrast requirements in Focus Appearance and Non-text
-                     Contrast is:
+                  <p>フォーカスの外観と非テキストのコントラスト要件の違いは、次のとおり:
                   </p>
                   
                   
                   <ul>
                      
-                     <li>Focus Appearance requires that focus indicators have a <em>change of contrast</em> between focused and non-focused states.
+                     <li>フォーカスの外観は、フォーカスインジケータが、フォーカスされた状態とフォーカスされていない状態の間で<em>コントラストが変化する</em>ことを要求する。
                      </li>
                      
-                     <li>Non-text Contrast requires that focus indicators have <em>adjacent contrast</em> between the indicator (in the focused state) and adjacent non-indicator colors.
+                     <li>非テキストコントラストでは、フォーカスインジケータは、インジケータ (フォーカス状態) と隣り合う非インジケータ色との間に<em>隣接コントラスト</em>を持つことが必要である。
                      </li>
                      
                   </ul>
                   
                   
                   <figure id="non-text-contrast-fail-focus-appearance-pass">
-                     <img src="img/focus-indicator-good-change-bad-adjacent.png" alt="A light blue button with a 3px thick border. The border is black in the unfocused state and light gray in the focused state." />
+                     <img src="img/focus-indicator-good-change-bad-adjacent.png" alt="太さ 3px のボーターを持つ水色のボタン。ボーターは、フォーカスされていない状態では黒で、フォーカスされている状態では薄いグレーになる。" />
                      
-                     <figcaption>Figure 18 This example passes Focus Appearance but fails Non-text Contrast; there is insufficient
-                        <em>adjacent contrast</em> between the focus indicator and the adjacent colors.
+                     <figcaption>図 18 この例は、フォーカスの外観は合格しているが、非テキストのコントラストは不合格である。フォーカスインジケータと隣接する色の間の<em>隣接コントラスト </em>が不十分である。
                      </figcaption>
                      
                   </figure>
                   
                   
                   <figure id="non-text-contrast-pass-focus-appearance-fail">
-                     <img src="img/focus-indicator-good-adjacent-bad-change.png" alt="A light blue button with a 3px thick border. The border is black in the unfocused state and dark gray in the focused state." />
+                     <img src="img/focus-indicator-good-adjacent-bad-change.png" alt="太さ 3px のボーターを持つ水色のボタン。ボーターは、フォーカスされていない状態では黒で、フォーカスされている状態では濃いグレーになる。" />
                      
-                     <figcaption>Figure 19 This example passes Non-text Contrast but fails Focus Appearance; there is insufficient
-                        <em>change of contrast</em> between the focused and unfocused states.
+                     <figcaption>図 19 この例は、非テキストコントラストは合格している、フォーカスの外観は不合格である。フォーカスされた状態とフォーカスされていない状態との間の<em>コントラストの変化</em>が不十分である。
```

Comment:
「非テキストコントラスト」は、達成基準名に合わせて「非テキストのコントラスト」としました。

```suggestion
                     <figcaption>図 19 この例は、非テキストのコントラストは合格している、フォーカスの外観は不合格である。フォーカスされた状態とフォーカスされていない状態との間の<em>コントラストの変化</em>が不十分である。
```